### PR TITLE
TCIOS-808835 Title string customisation capability for developers on SDK profile screen

### PR DESCRIPTION
**Pull request by sreedeepkesav**
Add support to set titles for APP's.
All titles are default specified and defaults to Login to

### CODE REVIEW

File: TrueSDK/External/TCTrueSDK.m [hunk 1]
```diff
@@ -41,6 +42,11 @@ - (void)setupWithAppKey:(nonnull NSString *)appKey
 {
     self.appKey = appKey;
     self.appLink = appLink;
+    self.titleType = TitleTypeDefault;
+}
+
+- (void)setTitleType:(enum TitleType)titleType {
```
Comment (elnaqah) [inline TrueSDK/External/TCTrueSDK.m]: do you need enum keyword here ?

File: TrueSDK/External/TCTrueSDK.m [hunk 2]
```diff
@@ -41,6 +42,11 @@ - (void)setupWithAppKey:(nonnull NSString *)appKey
...
+    self.titleType = titleType;
```
Comment (elnaqah) [inline TrueSDK/External/TCTrueSDK.m]: I think this is recursive, why do you need a setter?

Comment (elnaqah) [reply TrueSDK/External/TCTrueSDK.m]: self.titleType = titleType; call setTitleType

File: TrueSDK/External/TCTrueSDK.m (hunk 1 above)
Comment (sreedeepkesav) [reply TrueSDK/External/TCTrueSDK.m]: Done

File: TrueSDK/External/TCTrueSDK.m (hunk 2 above)
Comment (sreedeepkesav) [reply TrueSDK/External/TCTrueSDK.m]: Done.

**Pull request merged**